packages/test/test-end-to-end-tests/.eslintrc.cjs

```diff
@@ -14,6 +14,7 @@ module.exports = {
 
 		// This library is used in the browser, so we don't want dependencies on most node libraries.
 		"import/no-nodejs-modules": ["error", { allow: ["url"] }],
+		"import/namespace": "off",
```

Comment:
Can we add a comment documenting why this needs to be disabled?

Reply:
Fixed in latest.